src/main/java/com/aetherteam/aether_genesis/client/renderer/PhoenixDartLayer.java

```diff
@@ -0,0 +1,42 @@
+package com.aetherteam.aether_genesis.client.renderer;
+
+import com.aetherteam.aether.client.renderer.player.layer.AbstractDartLayer;
+import com.aetherteam.aether.entity.AetherEntityTypes;
+import com.aetherteam.aether.entity.projectile.dart.AbstractDart;
+import com.aetherteam.aether.entity.projectile.dart.GoldenDart;
+import com.aetherteam.aether_genesis.capability.player.GenesisPlayer;
+import com.mojang.blaze3d.vertex.PoseStack;
+import net.minecraft.client.model.PlayerModel;
+import net.minecraft.client.renderer.MultiBufferSource;
+import net.minecraft.client.renderer.entity.EntityRenderDispatcher;
+import net.minecraft.client.renderer.entity.LivingEntityRenderer;
+import net.minecraft.world.entity.Entity;
+import net.minecraft.world.entity.LivingEntity;
+import net.minecraft.world.entity.player.Player;
+
+import javax.annotation.Nonnull;
+import java.util.Optional;
+
+public class PhoenixDartLayer<T extends LivingEntity, M extends PlayerModel<T>> extends AbstractDartLayer<T, M> {
+    public PhoenixDartLayer(EntityRenderDispatcher renderDispatcher, LivingEntityRenderer<T, M> renderer) {
+        super(renderDispatcher, renderer);
+    }
+
+    @Override
+    protected int numStuck(@Nonnull T entity) {
+        if (entity instanceof Player player) {
+            Optional<GenesisPlayer> genesisPlayerOptional = GenesisPlayer.get(player).resolve();
+            if (genesisPlayerOptional.isPresent()) {
+                return genesisPlayerOptional.get().getGoldenDartCount();
```

Comment:
use `.getPhoenixDartCount()`

Reply:
sorry this is the first time i weite about capability related things, i'm so sorry for do this